spec/factories/snippet.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :first_snippet, class: 'Snippet' do
```

Comment:
You want a snippet factory, not enumerated factories that do the same thing.  Look at `sequence` in FactoryBot for this.